CHANGES.txt

```diff
@@ -5,7 +5,27 @@ CHANGELOG
 3.3.0 (unreleased)
 ==================
 
-- Nothing changed yet.
+- Cornice now only accepts Marshmallow schemas instead of instances
+- Marshmallow schemas get request object added to their context
+  during validation
+
+
+3.2.0 (2018-04-03)
+==================
+
+**New features**
+
+- Add marshmallow request validation support (#475, thanks @ergo)
+- Allow to reuse Pyramid existing routes in services (#477, thanks @ergo)
+
+**Documentation**
+
+- Make tutorial work for python3 users (#471, thanks @rvandegrift)
+
+**Internal changes**
+
+- Remove configuration commit (#476, thanks @ergo)
+- Remove mentions of ``text/json`` in tests and docs (#478, thanks @wjehenddher)
```

Comment:
Looks like you duplicated the 3.2.0 changelog entry

Reply:
Fixed, sorry